packages/neon-api/src/provider/dora/responses.ts

```diff
@@ -15,3 +16,10 @@ export interface DoraGetClaimableResponse {
   claimable: NeoCliClaimable[];
   unclaimed: number;
 }
+
+export type TEntry = Omit<Entry, "amount"> & { amount: number };
```

Comment:
What does `TEntry` mean?

Reply:
T means type and Entry means a unit of the entries field which is an array, so TENtry means the type of the entry information, basically

Reply:
Please follow the convention, prefix everything with `Dora` here to indicate that its the format returned from the API endpoint.